packages/devextreme/js/__internal/ui/chat/messagebubble.ts

```diff
@@ -50,6 +57,22 @@ class MessageBubble extends Widget<Properties> {
       return;
     }
 
+    if (isDeleted) {
+      const icon = $('<div>')
+        .addClass(ICON_CLASS)
+        .addClass(CHAT_MESSAGEBUBBLE_ICON_PROHIBITION_CLASS);
+
+      const deletedMessage = $('<div>')
+        .text(messageLocalization.format('dxChat-deletedMessageText'));
+
+      $bubbleContainer
+        .addClass(CHAT_MESSAGEBUBBLE_DELETED_CLASS)
```

Comment:
I assume that deleted class should be on root of message bubble or it should be inside content element. It depends on purposes of usign. Let's discuss this part if need